.github/workflows/set-versions.yml

```diff
@@ -0,0 +1,34 @@
+name: "Set versions"
+on:
+  workflow_dispatch:
+    inputs:
+      stableVersion:
+        description: 'Vert.x stable version, e.g 1.2.3'
+        required: true
+        type: string
+      stableSnapshotVersion:
+        description: 'Vert.x stable snapshot version, e.g. 1.2.4-SNAPSHOT'
+        required: true
+        type: string
+jobs:
+  set-versions:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
```

Comment:
@tsegismont removed the branch name there so it should use the current branch the workflow is applied to